config/initializers/content_security_policy.rb

```diff
@@ -25,6 +25,10 @@
     IdentityConfig.store.asset_host.presence,
   ].compact
 
+  if IdentityConfig.store.proofing_device_profiling_collecting_enabled
+    script_src << 'h.online-metrix.net'
+  end
+
```

Comment:
After talking with @zachmargolis, I realized that I gave some bad guidance here. Let's _not_ modify CSP in all environments: We want to ensure that we are not making calls to 3rd parties outside of production / staging. Apologies!

```suggestion
```

Reply:
That's my understanding as well

Reply:
I think we'll need the following:

- `script_src`
- `connect_src`
- `child_src`
- `image_src`

Reply:
(I'll leave a followup ticket for CSP stuff)

Reply:
Out of curiosity, why would `image_src` be needed?  I follow the other ones since we have tags for script/iframe and I presume the script will make calls (therefore `connect_src`).

Reply:
ThreatMetrix uses `<img>` tags as part of its profiling, loading up images sourced from `h.online-metrix.net`